phpbench.json

```diff
@@ -0,0 +1,49 @@
+{
+    "bootstrap": "vendor/symfony-cmf/testing/bootstrap/bootstrap.php",
+    "path": "Tests/Benchmark",
+    "reports": {
+        "index_overview": {
+            "extends": "aggregate",
+            "title": "Indexing Overview",
+            "description": "Comparison of indexing for each adapter overall",
+            "selector": ".//subject[group[@name='index']][parameter[@name='nb_documents']/@value = '1000']"
+        },
+        "index_scaling": {
+            "generator": "console_table",
+            "title": "Index scaling performance",
+            "headers": ["Adapter", "10 records", "100 records", "1000 records"],
+            "selector": ".//subject[group[@name='index']][parameter[@name='nb_documents']/@value = '1000']",
+            "cells": {
+                "adapter": "php:bench('class_name', string(ancestor::benchmark/@class))",
+                "10_records": "number(../subject[group[@name='index']][parameter[@name='nb_documents']/@value = '10']//iteration/@time)",
+                "100_records": "number(../subject[group[@name='index']][parameter[@name='nb_documents']/@value = '100']//iteration/@time)",
+                "1000_records": "number(../subject[group[@name='index']][parameter[@name='nb_documents']/@value = '1000']//iteration/@time)"
```

Comment:
Current objective is to figure out a way to make this sort of thing less verbose ..